crates/uv-build/src/lib.rs

```diff
@@ -732,22 +728,31 @@ impl SourceBuild {
             .map_or("None".to_string(), |path| {
                 format!(r#""{}""#, escape_path_for_python(path))
             });
+
+        // Write the hook output to a file so that we can read it back reliably.
+        let outfile = self
+            .temp_dir
+            .path()
+            .join(format!("build_{}.txt", self.build_kind));
```

Comment:
These don't _need_ deterministic names (we pass them into the script anyway) but I figured it would help debugging.